src/components/error-modal.cjsx

```diff
@@ -0,0 +1,76 @@
+# coffeelint: disable=max_line_length
```

Comment:
It's similar, but has a few differences.  This one attempts to render the HTML that exercises BE sends back on 500's, and has a different support link.  I considered extracting the Tutor error modal, but decided they were too different and relied to much on binding to a flux store.